packages/web-components/src/divider/divider.ts

```diff
@@ -51,13 +66,36 @@ export class Divider extends FASTElement {
   @attr
   public appearance?: DividerAppearance;
 
+  /**
+   * Handles changes to appearance attribute custom states
+   * @param prev - the previous state
+   * @param next - the next state
+   */
+  public appearanceChanged(prev: DividerAppearance | undefined, next: DividerAppearance | undefined) {
```

Comment:
This is an interesting pattern I don’t think I fully understand. Just a note for me to follow up when I’m back next week.